torch/_dynamo/variables/constant.py

```diff
@@ -103,6 +103,11 @@ def call_method(
                 items=self.unpack_var_sequence(tx), source=self.source, **options
             ).call_method(tx, name, args, kwargs)
 
+        if istype(self.value, list):
+            return variables.ListVariable(
+                items=self.unpack_var_sequence(tx), source=self.source, **options
+            ).call_method(tx, name, args, kwargs)
```

Comment:
it occurs to me that I'm not sure if it's a good idea to store lists in constant variables, because what if you mutate it 🤔

Reply:
(whatever problem this is, it's a preexisting condition)

Reply:
There's so many problems here. I narrowed it down with https://github.com/pytorch/pytorch/pull/105796 but I am working on another one at some point to lift the literals.